src/components/Dropdown/Dropdown.tsx

```diff
@@ -141,10 +143,12 @@ const DropdownComponent: FC<DropdownProps> = ({
       className={className}
       minWidth={buttonWidth}
     >
-      <TriggerWrapper setButtonWidth={setButtonWidth}>
-        {label}
-        {arrowIcon && <Icon className={theme.arrowIcon} />}
-      </TriggerWrapper>
+      {CustomTrigger ?? (
```

Comment:
What would happen if we don't pass the `CustomTrigger`?

Reply:
@paghar in this case, if you don't specify the "CustomTrigger" you don't have any trigger. Right? So, this is not correct.

Reply:
@rluders 
I use the default trigger. When there is no exist
 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing